mautic/mautic
# Upgraded Rector to minimum 0.11.54 and phpstan to 0.12.99

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [n ]
| New feature/enhancement? (use the a.x branch)      | [ y]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [n ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

The PR upgrade Rector to version to be able to use option `Option::APPLY_AUTO_IMPORT_NAMES_ON_CHANGED_FILES_ONLY `added in [0.11.54](https://github.com/rectorphp/rector/releases/tag/0.11.54)
Useful for refactor for Symfony 5


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11136"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
